CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- [FEATURE] Add support to batch processing within a transaction in OrionMySQLSink (#567)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/7d6c2c61abac91e24c4abbe0b3abb559bf10e765